utils/nativefuncs_test.go

```diff
@@ -149,6 +150,56 @@ func TestParseHelmChart(t *testing.T) {
 `)
 }
 
+func TestValidateJSONSchema(t *testing.T) {
+	vm := jsonnet.MakeVM()
+	RegisterNativeFuncs(vm, NewIdentityResolver())
+
+	res, err := vm.EvaluateSnippet("validObject", `
+    local schema = {
+      type: 'object',
+      properties: {
+        age: {
+          description: 'Age in years which must be equal to or greater than zero.',
+          type: 'integer',
+          minimum: 0,
+        },
+      },
+    };
+    local obj = {
+      age: 26,
+    };
+
+    std.native('validateJSONSchema')(obj, schema)`)
+	check(t, err, res, "true\n")
+
+	res, err = vm.EvaluateSnippet("invalidObjectErrors", `
+    local schema = {
+      type: 'object',
+      properties: {
+        projectName: {
+          description: 'projectName is required for the name of a project',
+          type: 'string',
+        },
+        language: {
+          description: 'Programming language implementation',
+          type: 'string',
+        },
+      },
+      required: ["projectName"],
+    };
+
+    local obj = {
+        language: 'go',
+    };
+
+    std.native('validateJSONSchema')(obj, schema)`)
+	if err != nil {
+		if !strings.Contains(err.Error(), "invalid against the schema") {
+			t.Errorf("expected invalid schema error. got: %s\n", err.Error())
+		}
```

Comment:
I'm not a huge fan of this, but I think it is _okay_ in a test. Any suggestions on this for a small test are appreciated :smile:

Reply:
it's ok